dallinger/recruiters.py

```diff
@@ -1,9 +1,8 @@
 """Recruiters manage the flow of participants to the experiment."""
-
 import os
 
 from boto.mturk.connection import MTurkConnection
-from psiturk.models import Participant
+from dallinger.models import Participant
```

Comment:
I don't think this can be swapped without a bit more work. Further down in this module there's a query using `Participant.hitid`, but dallinger's `Participant` model doesn't have that column (it has `hit_id`)